### DISCUSSION TITLE
Segmentation fault (core dumped) after running tests

### DISCUSSION THREAD

**Discussion by gxara** [pytest-dev/pytest#11521] [Q&A] [answered]
I've been facing challenges debugging a segmentation fault that has been occurring since October 3rd. The error surfaces after all tests have successfully completed. These tests are executed within an Azure Pipelines agent (Ubuntu 20.04)

The logs below were printed by using gdb [to get the stack trace ](https://codelv.com/blog/2021/12/debugging-ci-core-dump-with-gdb-and-pytest)
![image](https://github.com/pytest-dev/pytest/assets/33765053/e829cd12-1fed-4db0-a723-df60ea9b66f9)

**Comment by The-Compiler** [comment, marked as answer]
Might be this? Timing matches too, given that [greenlet 3.0.0](https://github.com/python-greenlet/greenlet/releases/tag/3.0.0) was released on October 2nd.

- https://github.com/python-greenlet/greenlet/issues/376

**Reply by gxara** [reply]
You nailed it!
Just pinned greenlet back to version 2.0.2
Thanks @The-Compiler
